qiita_pet/util.py

```diff
@@ -52,7 +52,7 @@ def clean_str(item):
     str
         cleaned string
     """
-    return str(item).replace(" ", "_").replace(":", "")
+    return str(item).replace(" ", "_").replace(":", "").replace(".", "")
```

Comment:
Are we sure that modifying this function will not impact assumptions made in other places in the codebase? Also, can you update the docstring for this function to reflect the code that's currently implemented (including your addition).